### DISCUSSION TITLE
Images and logos of Felizo

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#1] [Show and tell]
イメージ画像とロゴ画像です。discussion使用のトライも踏まえてアップします。
![image1](https://github.com/user-attachments/asse
![logo_1](https://github.com/user-attachments/assets/36996c51-4f94-49f8-9776-fc3a7ac866b4)
![logo_2](https://github.com/user-attachments/assets/b517c601-7d2f-4296-bcbe-efcbe8863385)
ts/3c36e46d-7647-4838-82e6-5ad6430fb4da)
![image2](https://github.com/user-attachments/assets/99a591a1-e05e-4513-9141-71344537f291)
![image3](https://github.com/user-attachments/assets/c303c325-2d60-4520-ac78-8cc0b0d168c8)

**Comment by tkg-geek** [comment]
上2ついいですね！
下2つわかるwww

**Comment by mikiIDE** [comment]
とにかくハッピーでポジティブな感じなの、好き！
